source/_includes/puppet-collections/_rsk_import_key.md

```diff
@@ -0,0 +1,6 @@
+    gpg: requesting key 4BD6EC30 from hkp server keys.gnupg.net
```

Comment:
Can you confirm to what versions of Puppet the new key applies? All versions? New versions? Specific versions?